charts/k8ez/values.yaml

```diff
@@ -36,8 +40,22 @@ podSecurityContext:
 # Enable deployment - by default we enable a simple deployment
 deployment: 
   enabled: true
-  # command: "sh"
-  # args: []
+  mainContainerNameOverride: ""
```

Comment:
Is there a reason for this?

Reply:
There was. I don't remember what it was off the top of my head. I'll dig around and try to remember.